packages/dev/core/src/Meshes/mesh.ts

```diff
@@ -2383,12 +2403,12 @@ export class Mesh extends AbstractMesh implements IGetSetVerticesData {
         if (
             !instanceDataStorage.isFrozen &&
             (this._internalMeshDataInfo._effectiveMaterial.backFaceCulling ||
-                this.overrideMaterialSideOrientation !== null ||
+                this.sideOrientation !== null ||

```

Comment:
Should be `=== null`, as now it is `Material.sideOrientation` which is optional and will be chosen if `sideOrientation === null`.